packages/message-forwarder/src/forwardMessage/forwardMessage.ts

```diff
@@ -21,6 +23,11 @@ const forwardMessage = async (
     return new Error(`Unsupported destination type: "${destinationType}"`)
   }
 
+  const conductorWorkflow = process.env.CONDUCTOR_WORKFLOW ?? "bichard_phase_1"
+  if (!conductorWorkflows.includes(conductorWorkflow)) {
+    return new Error(`Unsupported Conductor workflow: "${conductorWorkflow}"`)
+  }
+
   const aho = parseAhoXml(message)
```

Comment:
Should we parse both Aho and PncUpdateDataset?

Reply:
Good point

Reply:
I think this is just so we can extract the correlation ID for logging, maybe we should pull out the correlation ID extraction into its own function?

Reply:
Oooooooo yeahhh, that makes sense - alright if I do that in a separate PR?

Reply:
What about line 54?

return startBichardProcess(conductorWorkflow, aho, correlationId, conductorClient)
Does it start the new workflow with an AHO as input?

Reply:
It's safe to merge this PR. I just want to make sure when we use this for Phase 2, we don't miss PNC operations in the json.

Reply:
Hmmm, you're right Emad - not just for correlation ID as `startBichardProcess` adds the message to S3.

Reply:
Thanks for the spot!